### DISCUSSION TITLE
flusher_kafka_v2 发送消息时，部分特殊字符被转义成unicode编码

### DISCUSSION THREAD

**Discussion by leizhou666** [alibaba/ilogtail#847] [Help] [answered]
看了下代码，可能是跟Converter代码方法ConvertToSingleProtocolStream相关，里面用到了json.Marshal(log)
![image](https://github.com/alibaba/ilogtail/assets/97587488/8c98dcc9-c2b8-4971-824c-8271d37ce9d7)

输入：>>>><<<)(*&^%$#@!hello+1447138058167839254
输出：\u003e\u003e\u003e\u003e\u003c\u003c\u003c)(*\u0026^%$#@!hello+1447138058167839254

**Comment by messixukejia** [comment]
非常👍。是否可以尝试解决下？

**Reply by leizhou666** [reply]
好，我试试

**Comment by messixukejia** [comment, marked as answer]
PR：https://github.com/alibaba/ilogtail/commit/82ffc1556cf3d4a3921d86bc2e6747ce970084cf#diff-8c44644e92dacfdbb6a8ade0b00e98249dfebed9bf3dfe289f2a4f5d022424db
